doc/rst/language/spec/domains.rst

```diff
@@ -181,7 +177,7 @@ The syntax of a rectangular domain type is summarized as follows:
      'domain' ( named-expression-list )
 
 where ``named-expression-list`` permits the values of ``rank``,
-``idxType``, and ``stridable`` to be specified using standard type
+``idxType``, and ``strides`` to be specified using standard type
```

Comment:
The previous grammar of "... using standard type signature" seems a bit odd here. I think we could just get rid of that and have the sentence end with "to be specified".